src/docs/output-targets/custom-elements.md

```diff
@@ -75,20 +68,42 @@ export const config: Config = {
 | `auto-define-custom-elements` | A component and its children will be automatically defined with the `CustomElementRegistry` when the component's module is imported.                                                                                                                                                                                                                                                                      |
 | `single-export-module`        | All component and custom element definition helper functions will be exported from the `index.js` file in the output directory (see [Defining Exported Custom Elements](#defining-exported-custom-elements) for more information on this file's purpose). This file can be used as the root module when distributing your component library, see [below](#distributing-custom-elements) for more details. |
 
-> Note: At this time, components created not using JSX may not be automatically
-> defined. This is a known limitation of the API and users should be aware of
-> it
+> Note: At this time, components created not using JSX may not be automatically defined. This is a known limitation of the API and users should be aware of it.
```

Comment:
Looks like @alicewriteswrongs has a PR open to change this block

Reply:
I just thought it sounded a bit odd!

Reply:
It did. My bad 🙃

Reply:
Since this blockquote has now been changed on `main` should we merge `main` into the v3 branch and rebase this PR over that to make sure things are consistent? I think it would be easy to lose a change later in a merge commit somewhere.

Reply:
Yah - I'll do that after we merge this into v3 (there're already conflicts there, and I'd like to do them in one fell swoop)